journals/journal_abbreviations_acs.csv

```diff
@@ -75,7 +78,6 @@ Annales Pharmaceutiques Francaises;Ann. Pharm. Fr.
 Annales de Chimie (Cachan, France);Ann. Chim. (Cachan, Fr.)
 Annals of Nuclear Energy;Ann. Nucl. Energy
 Annals of Nutrition & Metabolism;Ann. Nutr. Metab.
-Annals of Physics (Amsterdam, Netherlands);Ann. Phys. (Amsterdam, Neth.)
```

Comment:
Could you comment on the removed entries?

Reply:
Annals of Physics is officially published in New York. I guess, as Elsevier's HQ are in Amsterdam, it is commonly misreferenced as Annals of Physics (Amsterdam, Netherlands). See e.g. https://en.wikipedia.org/wiki/Annals_of_Physics

Should I still leave it in the list?

Reply:
By the way, there are several articles in the list with multiple abbreviations. Should I throw out all the incorrect ones, or put them in the last position (so that unabbreviate works also for wrong abbreviations)?

Reply:
Please remove it from the list. It would be great if you added another list with wrong abbreviations. Have to think longer how to handle them. -- I am aiming for having correct journal lists here, which can be then also be used by other tools. The lists themselves should be intuitive. Having only correct ones in a list seems to be more intuitive than having correct ones and "fixing ones".